cobalt/build/gn.py

```diff
@@ -56,7 +56,8 @@ def get_build_args(build_args_path):
     },
     'qa': {
         'symbol_level': 1,
-        'is_debug': 'false'
+        'is_debug': 'false',
```

Comment:
Nit: I think this should be removed if you're adding `is_official_build` here.

Reply:
Done.